care.js

```diff
@@ -3,13 +3,13 @@ var config = require(__dirname + '/config.js');
 var twitterbot = require(__dirname + '/twitterbot.js');
 var gitbot = require(__dirname + '/gitbot.js');
 var pomodoro = require(__dirname + '/pomodoro.js');
+var ansiArt = require('ansi-art').default;
 
 var notifier = require('node-notifier');
 var spawn = require('child_process').spawn;
 var blessed = require('blessed');
 var contrib = require('blessed-contrib');
 var chalk = require('chalk');
-var parrotSay = require('parrotsay-api');
```

Comment:
Yup, it's moved to ansi-art here: https://github.com/gauravchl/ansi-art/tree/master/src/arts
It's almost same except the speech bubble is shifted a pixel above:

![group](https://cloud.githubusercontent.com/assets/3471415/26141419/55daa544-3af9-11e7-81f8-16d0853847b1.jpg)
This make speech bubble renders on all other arts without overlapping the art pixels itself.
Or should use the old one(parrot-say)!?